ex03.ex

```diff
@@ -77,7 +110,19 @@ defmodule Ex03 do
 
   """
 
-  def list_contains . .. "your code"
+    def list_contains([],_val) do false end
```

Comment:
### -4

Again, really complicated code for something that's simple:

~~~ elixir
  def list_contains([], _value),             do: false
  def list_contains([value | _rest], value), do: true
  def list_contains([_ | rest], value),      do: list_contains(rest, value)
~~~

Try to develop a feeling for complexity. When you're coding and you're having to code lots of extra flags and stuff, get yourself to stop and think.

Sometimes code has to be complex. But often code can be simpler than you think.